dev/devicelab/bin/tasks/platform_services__button_tap.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
`button_tap` is too much detail for a task. We want the task to be coarse-grained. Let's call it `platform_services_test` and encapsulate the specifics.